portfolio/src/main/webapp/script.js

```diff
@@ -75,8 +48,49 @@ function displayComments(comments) {
  * Sends request to remove all comments
  */
 function deleteAllComments() {
+
+  window.confirm("Are you sure you want to delete all the comments?");
+  
+  var confirmpss = "911b0a07a8cacfebc5f1f45596d67017136c950499fa5b4ff6faffa031f3cec7f197853d1660712c154e1f59c60f682e34ea9b5cbd2d8d5adb0c834f963f30de";
```

Comment:
Use const instead of var here.

Try not to use ```pss``` as an abbreviation.

>Give as descriptive a name as possible, within reason. Do not worry about saving horizontal space as it is far more important to make your code immediately understandable by a new reader. Do not use abbreviations that are ambiguous or unfamiliar to readers outside your project, and do not abbreviate by deleting letters within a word.

([Documentation](https://google.github.io/styleguide/jsguide.html#naming-rules-common-to-all-identifiers))